src/base/gps.cc

```diff
@@ -38,23 +38,20 @@ namespace colmap {
 GPSTransform::GPSTransform(const int ellipsoid) {
   switch (ellipsoid) {
     case GRS80:
-      a_ = 6378137;
-      b_ = 6.356752314140356e+06;
-      f_ = 0.003352810681182;
-      break;
     case WGS84:
-      a_ = 6378137;
-      b_ = 6.356752314245179e+06;
-      f_ = 0.003352810664747;
+      a_ = 6378137.0;
+      // Using flattening factor and semi-minor axis calculation from Bing maps.
+      // New calculations are marginally more accurate than the original values.
+      f_ = 1.0 / 298.257222101;
```

Comment:
Accurate f of WGS84 is 1.0 / 298.257223563.  You are using a wrong value here,which comes from GRS80(1.0 / 
 298.2572221008827112431628366). Please refer to https://en.wikipedia.org/wiki/World_Geodetic_System

Reply:
@Freeverc Thanks for detecting this issue. Would you be able to submit a fix to revert back ?

Reply:
That's ok. Fix submited.